src/runtime/FunctionObject.cpp

```diff
@@ -342,7 +337,7 @@ Value FunctionObject::processCall(ExecutionState& state, const Value& receiverSr
     }
 
     if (UNLIKELY(isSuperCall && isBuiltin() && !isNewExpression)) {
```

Comment:
For readability / State expression explicitly
if (UNLIKELY(isSuperCall == true && isBuiltin() == true && isNewExpression == false))